src/environments/environment.ts

```diff
@@ -7,6 +7,10 @@ import 'zone.js/dist/zone-error';
 export const environment = {
   production: false,
   siteName: 'CXCloud DEV',
+  commerceApiUrl: 'https://dev.cxcloud.com/api/v1',
```

Comment:
Also would do it in more explicit way in case if you need to add some extra config per service:
```ts
commerce: {
 apiUrl: 'https://.....'
}
```